backend/stakepoold/server.go

```diff
@@ -77,7 +76,7 @@ func calculateFeeAddresses(xpubStr string, params *chaincfg.Params) (map[string]
 	}
 
 	// Derive from external branch
```

Comment:
Since we lose this information without the name, could you either include this comment in all affected places, or remake the constant in helpers package?

Reply:
Agreed, a constant instead of magic numbers would be nice please.